content/en/docs/ambient/install/platform-prerequisites/index.md

```diff
@@ -1,73 +1,178 @@
     {{< /text >}}
 
-### K3S
+{{< /tab >}}
+
+{{< tab name="istioctl" category-value="istioctl" >}}
+
+    {{< text syntax=bash >}}
+    $ istioctl install --set profile=ambient --set values.cni.cniConfDir=/var/snap/microk8s/current/args/cni-network --set values.cni.cniBinDir=/var/snap/microk8s/current/opt/cni/bin
+    {{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+### minikube
+
+If you are using [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) with the [Docker driver](https://minikube.sigs.k8s.io/docs/drivers/docker/),
+you must append some values to your installation command so that the Istio CNI node agent can correctly manage
+and capture pods on the node. For example:
+
+{{< tabset category-name="install-method" >}}
+
+{{< tab name="Helm" category-value="helm" >}}
+
+    {{< text syntax=bash >}}
+    $ helm install istio-cni istio/cni -n istio-system --set profile=ambient --wait --set cniNetnsDir="/var/run/docker/netns"
+    {{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="istioctl" category-value="istioctl" >}}
+
+    {{< text syntax=bash >}}
+    $ istioctl install --set profile=ambient --set cni.cniNetnsDir="/var/run/docker/netns"
+    {{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+### Red Hat OpenShift
+
+OpenShift requires that `ztunnel` and `istio-cni` components are installed in the `kube-system` namespace.  An `openshift-ambient` installation profile is provided which will make this change for you.  Replace instances of `profile=ambient` with `profile=openshift-ambient` in the installation commands. For example:
+
+{{< tabset category-name="install-method" >}}
+
+{{< tab name="Helm" category-value="helm" >}}
+
+    {{< text syntax=bash >}}
+    $ helm install istio-cni istio/cni -n istio-system --set profile=openshift-ambient --wait
+    {{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="istioctl" category-value="istioctl" >}}
+
+    {{< text syntax=bash >}}
+    $ istioctl install --set profile=openshift-ambient --skip-confirmation
+    {{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
 
-1. If you are using [K3S](https://k3s.io/) and one of its bundled CNIs, you must append `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/var/lib/rancher/k3s/data/current/bin/` to your `istioctl install` or `helm install` command to install Istio ambient, as K3S uses nonstandard locations for CNI configuration and binaries. These nonstandard locations may be overridden as well [according to K3S documentation](https://docs.k3s.io/cli/server#k3s-server-cli-help). If you are using K3S with a custom, non-bundled CNI, you must use the correct paths for those CNIs, e.g. `/etc/cni/net.d` - [see K3S docs for details](https://docs.k3s.io/networking/basic-network-options#custom-cni).
+## CNI plugins
 
-## CNI
+The following configurations apply to all platforms, when certain {{< gloss "CNI" >}}CNI plugins{{< /gloss >}} are used:
```

Comment:
It would be more correct to say "[CNI runtime](https://github.com/containernetworking/cni/blob/main/SPEC.md#overview)"?  But less understood, I would think. [Kubernetes uses the term 'CNI plugin'](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/), as do many of the people that make them:

> Kubernetes uses the Container Network Interface (CNI) to interact with networking providers like Calico. The Calico binary that presents this API to Kubernetes is called the CNI plugin and must be installed on every node in the Kubernetes cluster.

and thus I think this is a fight that is lost outside of the universe of people who actually write CNI implementations.

We were trying to have our thing be called the 'CNI node agent' to differentiate it.  Is that not sensible?

(p.s. I see you)

<img width="1454" alt="Screenshot 2024-08-21 at 12 16 44 PM" src="https://github.com/user-attachments/assets/98400170-ed08-4767-82d9-b237011a1545">

Reply:
`CNI node agent` is the most sensible, I mostly just don't want to say `CNI plugin` and have it not be clear if we're talking about our thing (which is not a CNI, and consists of more than a plugin) or the thing you already have (which is, and consists of more than a plugin).